velox/dwio/common/FilterNode.h

```diff
@@ -102,23 +102,23 @@ struct FilterNode {
     return name == other.name;
   }
 
-  bool match(const std::string_view& name) const {
+  bool match(const std::string_view& name_to_match) const {
```

Comment:
Variable name should be camelCase according to the code style? https://github.com/facebookincubator/velox/blob/main/CODING_STYLE.md#naming-conventions